config/src/main/java/com/alibaba/nacos/config/server/controller/CapacityController.java

```diff
@@ -122,19 +121,19 @@ public CapacityController(CapacityService capacityService) {
         RestResult<Boolean> restResult = new RestResult<Boolean>();
         if (StringUtils.isBlank(targetFieldValue)) {
             setFailResult(response, restResult, 400);
-            restResult.setMessage(String.format("参数%s为空", targetFieldName));
+            restResult.setMessage(String.format("The parameter %s is null", targetFieldName));
             return restResult;
         }
         try {
             boolean insertOrUpdateResult = capacityService
                     .insertOrUpdateCapacity(group, tenant, quota, maxSize, maxAggrCount, maxAggrSize);
             if (insertOrUpdateResult) {
                 setSuccessResult(response, restResult);
-                restResult.setMessage(String.format("成功更新%s为%s的容量信息配置", targetFieldName, targetFieldValue));
+                restResult.setMessage(String.format("Successfully updated %s for capacity information configuration for %s", targetFieldName, targetFieldValue));
```

Comment:
The configuration information of %s has been successfully updated to %s

Reply:
0xa36fD8fF0524f287422337d99bABdf05923f581B

Reply:
https://app.circleci.com/pipelines/github/MBANKm/mkdocs-windmill?invite=true